<launch>
<!-- use argument for choosing yaml file for initial control-->
<!-- roslaunch traj_action_client.launch yaml:= ?init_control? -->
 <arg name="control" default="3mps_4" />
  <!-- <arg name="control" default="drift5" /> This is the parallel parking one.-->
  <!-- <arg name="control" default="aum3" /> --> <!-- This is the 1m one. --> 

<!-- Load Planner configs from YAML -->
  <rosparam command="load" file="$(find ilqr_loco)/config/bsilqr_params_fve.yaml"/>
  <rosparam command="load" file="$(find ilqr_loco)/control_seq/$(arg control).yaml"/>

<!-- Start planner client-server nodes-->
  <node pkg="ilqr_loco" type="traj_client" name="traj_client" output="screen"/>
  <node pkg="ilqr_loco" type="traj_server" name="traj_server" output="screen"/>

</launch>
